lib/galaxy/datatypes/text.py

```diff
@@ -84,6 +84,44 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
         return False
 
 
+@build_sniff_from_prefix
+class Zarr(Text):
+    """Class describing a zarr dataset"""
+    
+    edam_format = "format_3915"
+    file_ext = "zarr"
+
```

Comment:
should we at least add a metadata element with a `version`? Similar to other datatypes?